packages/react-native-bridge/android/build.gradle

```diff
@@ -341,3 +379,64 @@ If they are changed, the isBundleUpToDate flag is switched to false. That flag i
         delete nodeModulesFolders
     }
 }
+
+// Allows to override the artifact version by passing a -PbintrayVersion
+// parameter when calling Gradle
+def getBintrayVersion() {
+  return project.properties['bintrayVersion'] ?: android.defaultConfig.versionName
+}
+
+bintray {
+    user = project.hasProperty('bintrayUser') ? project.property('bintrayUser') : System.getenv('BINTRAY_USER')
+    key = project.hasProperty('bintrayKey') ? project.property('bintrayKey') : System.getenv('BINTRAY_KEY')
+    publications = ['GutenbergMobilePublication']
+    publish = true
+    pkg {
+        repo = 'maven'
+        name = 'react-native-gutenberg-bridge'
+        licenses = ['GPL-2.0']
+        // FIXME: Once this works properly poit to 'wordpress-mobile'
+        userOrg = 'mokagio'
+        // FIXME: Once this works properly poit to the original gutenberg (or gutenberg-mobile) repo. See also: https://github.com/oguzkocer/gutenberg/commit/0ca71b1abe3ea22b77abb3f390534e66e39161d5#diff-320e7dacc9e9a83ac8128d299319f82cf6dfa2609e0ebbc043d13de33ea92773R350-R353
+        vcsUrl = 'https://github.com/wordpress-mobile/gutenberg-mobile.git'
```

Comment:
I discussed this with @oguzkocer and @mchowning. I'll update to point to `wordpress-mobile` before merging this.